public/components.html

```diff
@@ -0,0 +1,588 @@
+                    </div>
+                </div>
+            </div>
+
+            <hr>
+
+            <!-- Cards -->
+            <div id="cards">
+                <h1 class="card-heading ">Cards</h1>
+                <div class="offwhite-card ">
+                    <h2 class="sub-heading ">Checkout Card <button class="myBtn_multi code-btn"><i class="fas fa-code "></i></button></h2>
+                    <div class="product-card ">
+                        <div class="product-img "></div>
+                        <div class="product-desc ">
+                            <div class="product-information ">
+                                <h2 style="font-weight: lighter ">LI-NING XP-90-IV Black, White, Silver</h2>
+                                <span>Delivery in 3-4 days | </span>
+                                <span style="color:green;font-weight:bold ">Free</span>
+                                <s>₹40</s>
+                                <p>Pack of:1, 86 g</p>
+                                <div style="display: flex; ">
+                                    <p style="margin-right:10px ">Seller: RetailNet </p>
+                                    <img height="20px " src=" ../images/fassured.png " alt=" ">
+                                </div>
+                                <h3>₹350</h3> <span> <s>₹990</s></span>
+                                <strong style="color:green ">64% Off 1 offer available </strong>
+                                <br>
+                                <h2 style="display:block; font: size 1rem;margin-top:1.4rem; ">REMOVE</h2>
+                            </div>
+                        </div>
+                        <div class=" product-regulator ">
+                            <button>-</button>
+                            <span>2</span>
+                            <button>+</button>
+                        </div>
+                    </div>
+                    <div class="product-card ">
+                        <div class="product-img "></div>
+                        <div class="product-desc ">
+                            <div class="product-information ">
+                                <h2 style="font-weight: lighter ">LI-NING XP-90-IV Black, White, Silver</h2>
+                                <span>Delivery in 3-4 days | </span>
+                                <span style="color:green;font-weight:bold ">Free</span>
+                                <s>₹40</s>
+                                <p>Pack of:1, 86 g</p>
+                                <div style="display: flex; ">
+                                    <p style="margin-right:10px ">Seller: RetailNet </p>
+                                    <img height="20px " src=" ../images/fassured.png " alt=" ">
+                                </div>
+                                <h3>₹350</h3> <span> <s>₹990</s></span>
+                                <strong style="color:green ">64% Off 1 offer available </strong>
```

Comment:
use small tag instead of strong . It will look cool in small tag

Reply:
Actually, I used the strong tag to explicitly highlight the offer to the user.